### PR TITLE
#1121 Support multiple type devices

### PR DESCRIPTION
**Pull request by lizhiboo**
**What this PR does / why we need it**:
#1121
**Which issue(s) this PR fixes** _(optional, in `Fixes #<issue number>, #<issue number>, ...` format, will close the issue(s) when PR gets merged)_:
Fixes #

### CODE REVIEW

File: charts/custom-serving/values.yaml [hunk 1]
```diff
@@ -14,6 +14,9 @@ restApiPort: 8501
 metricsPort: 0
 replicas: 1
 
+# device resources
+#devices: hygon.com/dcu=1
```
Comment (cheyang) [inline charts/custom-serving/values.yaml:18]: I suggest using `amd.com/gpu` instead.

**Comment by Syulin7** [comment]
@lizhiboo Thank you for this great contribution. Please rebase your PR

File: sdk/arena-java-sdk/src/test/java/com/github/kubeflow/arena/PyTorchJobTest.java [hunk 2]
```diff
@@ -19,13 +19,17 @@ public void testSubmit() throws ArenaException, IOException {
...
+        labels.put("arena.kubeflow.org/console-user", "1234567890");
+
         String command = "python /root/code/mnist-pytorch/mnist.py --backend gloo";
 
         TrainingJob job = new PytorchJobBuilder()
                 .ttlAfterFinished("30s")
                 .name("test-pytorchjob-1")
                 .gpus(1)
                 .labels(labels)
+                .devices(labels)
```
Comment (Syulin7) [inline sdk/arena-java-sdk/src/test/java/com/github/kubeflow/arena/PyTorchJobTest.java:32]: it is devices instead of labels?

**Comment by Syulin7** [comment]
/lgtm
cc @cheyang

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**